charts/substra-backend/values.yaml

```diff
@@ -203,6 +203,7 @@ docker-registry:
 celerybeat:
   replicaCount: 1
   expiredTokensFlushPeriod: 86400
+  maximumImagesTTL: 86400
```

Comment:
This is not the same as the default in celery.py (7 * 24 * 3600 = 604800), is it on purpose?

Reply:
Yep, I don't know what to put as default value in the chart  (24h vs 7 days)

Reply:
7 days seems more conservative, that would be a sound default.

Reply:
Done